bindings/python/pyproject.toml

```diff
@@ -32,6 +33,7 @@ classifiers = [
   "Programming Language :: Python :: 3.10",
   "Programming Language :: Python :: 3.11",
   "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
```

Comment:
nit: technically 3.13 is not officially supported
https://github.com/apache/iceberg-python/blob/b15934d5a9e6bf97b047f63239cc21ba1c15cdd4/pyproject.toml#L26-L33

Reply:
"supported" here means official support from pyiceberg. we allow people to use 3.13 for testing. I think adding `3.13` here might signal that 3.13 can be used officially